src/chunk.zig

```diff
@@ -413,6 +424,16 @@ pub const ServerChunk = struct { // MARK: ServerChunk
 		}
 	}
 
+	/// Updates a block unconditionally.
+	pub fn updateBlock(self: *ServerChunk, _x: i32, _y: i32, _z: i32, newBlock: Block) void {
```

Comment:
I think I need that one as I need to check few different current block conditions.

Reply:
Does updateBlockInGeneration not work?

Reply:
Oh, its identical XD
I thought its something different since it has this `InGeneration` suffix and didn't even look into implementation.